docs/book/src/clusterctl/commands/move.md

```diff
@@ -35,3 +35,29 @@ The `Cluster` object created in the target management cluster instead will be ac
 process completes. 
 
 </aside>
+
+## Pivot
+
+Pivot is a process for moving the provider components and declared cluster-api resources from a source management
+cluster to a target management cluster.
+
+This can now achieved with the following procedure:
+
+1. Use `clusterctl init` to install the provider components into the target management cluster
+2. Use `clusterctl move` to move the cluster-api resources from a Source Management cluster to a Target Management cluster
+
+## Bootstrap & Pivot
+
+In former versions of clusterctl the pivot process was bounded with the creation of a temporary bootstrap cluster
+ used to provision a target Management cluster.
+
+This can now achieved with the following procedure:
```

Comment:
Maybe remove the section on the old clusterctl behaviour as far as this doc goes.

Reply:
@randomvariable I have changed the sentence so there is no more reference to old versions